### PR TITLE
feat(playground): use real code sample

### LINKED ISSUE #4769: Cant modify playground text
Yeah just cant modify ant text on the playground regardless of browser or extensions.

**Prettier 1.13.7**
[Playground link](https://prettier.io/playground/)
```sh
--parser babylon
```

**Input:**
```jsx
function HelloWorld({greeting = "hello", greeted = '"World"', silent = false, onMouseOver,}) {

  if(!greeting){return null};

     // TODO: Don't use random in render
  let num = Math.floor (Math.random() * 1E+7).toString().replace(/\.\d+/ig, "")

  return <div className='HelloWorld' title={`You are visitor number ${ num }`} onMouseOver={onMouseOver}>

    <strong>{ greeting.slice( 0, 1 ).toUpperCase() + greeting.slice(1).toLowerCase() }</strong>
    {greeting.endsWith(",") ? " " : <span style={{color: '\grey'}}>", "</span> }
    <em>
	{ greeted }
	</em>
    { (silent)
      ? "."
      : "!"}

    </div>;

}
```

**Output:**
```jsx
function HelloWorld({
  greeting = "hello",
  greeted = '"World"',
  silent = false,
  onMouseOver
}) {
  if (!greeting) {
    return null;
  }

  // TODO: Don't use random in render
  let num = Math.floor(Math.random() * 1e7)
    .toString()
    .replace(/\.\d+/gi, "");

  return (
    <div
      className="HelloWorld"
      title={`You are visitor number ${num}`}
      onMouseOver={onMouseOver}
    >
      <strong>
        {greeting.slice(0, 1).toUpperCase() + greeting.slice(1).toLowerCase()}
      </strong>
      {greeting.endsWith(",") ? (
        " "
      ) : (
        <span style={{ color: "grey" }}>", "</span>
      )}
      <em>{greeted}</em>
      {silent ? "." : "!"}
    </div>
  );
}

```

**Expected behavior:**

### PR DESCRIPTION
**Pull request by ikatyang**
Closes #4769

[preview playground](https://deploy-preview-4770--prettier.netlify.com/playground/)

### CODE REVIEW

File: website/playground/panels.js [hunk 1]
```diff
@@ -150,7 +159,7 @@ export function InputPanel(props) {
       autoCloseBrackets={true}
       matchBrackets={true}
       showCursorWhenSelecting={true}
-      tabWidth={2}
+      tabSize={4}
```
Comment (ikatyang) [inline website/playground/panels.js]: http://codemirror.net/doc/manual.html#option_tabSize

Not sure if it should be 2 or 4.

**Comment by lydell** [comment]
People wouldn't know to double-click, though?

**Comment by ikatyang** [comment]
I thought people would random click it if they think it's editable but not able to edit it. 🤔

**Comment by hawkrives** [comment]
How about if it's just actual content by default? I'm always confused that it's a placeholder and I have to remind myself that I don't have to clear it out.

**Comment by ikatyang** [comment]
Then there will be no code example for other languages, see #3652 for why placeholder exists.

**Comment by duailibe** [comment]
> Then there will be no code example for other languages

I don't understand this. What does this have to do with the placeholder?

**Comment by hawkrives** [comment]
@ikatyang yeah, and I've been confused by it ever since 😜

As an actual reason to switch to a code sample, it'll allow a potential user to experiment by editing the sample and watch how the format changes from the sample text.

**Comment by lydell** [comment]
I think it's better to go back to real text again, as long as we have an easy to find Clear button.

When switching language we could do:

```
if content == oldLanguage.placeholder:
    content = newLanguage.placeholder
```

**Comment by lydell** [comment]
I like it!

I'm just a little bit worried that people won't see the Clear button.

Does it make sense to automatically select all text when focusing the text area? (Perhaps only if it contains sample code?) That one can paste or start typing without having to clear first.

@j-f1 since you requested the placeholder thing in #3652 – what do you think?

**Comment by j-f1** [comment]
The concept seems good, but anything I type comes out backwards :/ (on an iPhone)

**Comment by duailibe** [comment]
Yeah it's prepending text when typing

File: website/playground/panels.js [hunk 2]
```diff
@@ -38,6 +38,9 @@ class CodeMirrorPanel extends React.Component {
     if (this.props.readOnly && this.props.value !== this._cached) {
       this.updateValue(this.props.value);
     }
+    if (this.props.value !== prevProps.value) {
+      this.updateValue(this.props.value);
+    }
```
Comment (duailibe) [inline website/playground/panels.js]: I think you can just remove the `this.props.readOnly` above

**Comment by lydell** [comment]
Thoughts on my auto-select-text idea?

**Comment by ikatyang** [comment]
Feels not bad, though it doesn't work on mobile.

**Comment by j-f1** [comment]
Sounds reasonable @lydell.

**Pull request merged**